.github/workflows/docs-configuration-reference.yml

```diff
@@ -0,0 +1,73 @@
+name: Verify docs
+
+on:
+  schedule:
+    - cron: '0 10 * * *'
+
+jobs:
+  use:
+    name: Configuration reference
+    runs-on: Ubuntu-20.04
+    strategy:
+      fail-fast: false
+
+    steps:
+      - name: Set up PHP
+        uses: shivammathur/setup-php@2.7.0
+        with:
+          php-version: 7.4
+          coverage: none
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Get composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: composer-${{ runner.os }}-7.4-${{ hashFiles('composer.*') }}
+          restore-keys: |
+            composer-${{ runner.os }}-7.4-
+            composer-${{ runner.os }}-
+            composer-
+
+      - name: Checkout Symfony repo
+        run: composer create-project --stability dev symfony/website-skeleton .github/workflows/docs-configuration-reference/symfony
+
+      - name: Checkout Symfony Docs repo
+        run: git clone https://github.com/symfony/symfony-docs .github/workflows/docs-configuration-reference/docs
+
+      - name: Download dependencies
+        run: composer install --no-interaction --optimize-autoloader
+
+      - name: Verify docs
+        env:
+          GITHUB_TOKEN: ${{ secrets.CARSONPROD_GITHUB_TOKEN }}
```

Comment:
This secret needs to be added

Reply:
Maybe it has been added already?

Reply:
Yes, this does exist already on this repository

Reply:
Thank you